plugins/process.py

```diff
@@ -174,7 +189,7 @@ def generate_tables(self, temp_dir, exclude_dnp):
             #     2: 'smt'
             # }.get(footprint.GetAttributes())
 
-            skip_footprint = exclude_dnp and (footprint.HasProperty('dnp') or footprint.GetValue().upper() == 'DNP')
+            skip_footprint = exclude_dnp and (footprint.HasFieldByName('dnp') or footprint.GetValue().upper() == 'DNP')
```

Comment:
shouldn't that be `skip_footprint = exclude_dnp and (footprint_has_field(footprint, 'dnp') or footprint.GetValue().upper() == 'DNP')`?

Reply:
Thank you for finding that little bug.